packages/expo-local-authentication/src/LocalAuthentication.types.ts

```diff
@@ -1,4 +1,4 @@
-export type LocalAuthenticationResult = { success: true } | { success: false; error: string };
+export type LocalAuthenticationResult = { success: true } | { success: false; error: string; message: string };
```

Comment:
```suggestion
export type LocalAuthenticationResult = { success: true } | { success: false; error: string; warning?: string };
```
It should be called a warning and should be optional.